<ui version="4.0" >
 <class>DriverShiftDialog</class>
 <widget class="QDialog" name="DriverShiftDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Driver work shift</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Parking:*</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QComboBox" name="parkingscomboBox" />
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Driver:*</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="driverlineEdit" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>324</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="closepushButton" >
         <property name="text" >
          <string>Close</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okpushButton" >
         <property name="text" >
          <string>Save</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelpushButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>driverlineEdit</tabstop>
  <tabstop>parkingscomboBox</tabstop>
  <tabstop>okpushButton</tabstop>
  <tabstop>cancelpushButton</tabstop>
  <tabstop>closepushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelpushButton</sender>
   <signal>clicked()</signal>
   <receiver>DriverShiftDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>237</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel" >
     <x>300</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
